ios/FluentUI/Button/FluentButtonStyle.swift

```diff
@@ -0,0 +1,98 @@
+//
+//  Copyright (c) Microsoft Corporation. All rights reserved.
+//  Licensed under the MIT License.
+//
+
+import SwiftUI
+import UIKit
+
+/// ButtonStyle which configures the Button View according to its state and design tokens.
+public struct FluentButtonStyle: SwiftUI.ButtonStyle {
+    public init(style: ButtonStyle, size: ButtonSizeCategory) {
+        self.style = style
+        self.size = size
+        self.tokenSet = ButtonTokenSet(style: { style }, size: { size })
+    }
+
+    public func makeBody(configuration: Configuration) -> some View {
+        tokenSet.update(fluentTheme)
```

Comment:
Shouldn't the token set just be transient thing that we make using the fluent theme here in the body? When do we ever need the observation?

Reply:
As mentioned in a previous comment reply, this is a carry-over from when this was shared with a parent wrapper class. Now that it's unnecessary, we can simplify the logic a bit and skip storing the class at all.